### ISSUE TITLE
Error while trying to convert webpage from remote URL (Error with steps to reproduce)

### ISSUE THREAD

**Issue by wildeyes** [bjrmatos/electron-html-to#457]
Using latest version of electron html to and express, I wrote a simple example using express and the `electron-html-to` tutorial.

```
import * as express from 'express';
import * as convertFactory from 'electron-html-to';
import * as fs from 'fs';

const app = express();

app.get('*', (req, res) => {
  const hasUrl = req.params && req.params[0];
  if (hasUrl) {
    const urlWithSlash = req.params[0];
    const url = urlWithSlash.slice(1);
    const conversion = convertFactory({
      converterPath: convertFactory.converters.PDF
    });
    console.log('starting converting!');
    conversion({ url }, (err, result) => {
      if (err) return console.error('Conversion failed with error:', err);
      // TODO using url as the name of the file - dangerous.
      const filename = url + '.pdf';
      result.stream.pipe(fs.createWriteStream(filename));
      console.log('done converting!')
      // res.download(filename);
      // conversion.kill();
    })
  }
});

app.listen(process.env.PORT || 1337, () => console.log('PDF server listening on', process.env.PORT || 1337))
```

I am running the example with `ts-node`, and then I `curl "localhost:1337/https://expressjs.com/en/starter/hello-world.html"`.

I expect to see a "https://expressjs.com/en/starter/hello-world.html.pdf` file in my project's folder, but instead I'm getting
```
PDF server listening on 1337
starting converting!
(node:18524) DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
Conversion failed with error: Error: An error has occurred when trying to process the task
    at ElectronWorker.onWorkerProcessIpcMessage (C:\Users\Documents\node-print-to-pdf\node_modules\electron-workers\lib\ElectronWorker.js:193:25)
    at emitTwo (events.js:111:20)
    at ChildProcess.emit (events.js:194:7)
    at process.nextTick (internal/child_process.js:766:12)
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickDomainCallback (internal/process/next_tick.js:128:9)
```

Have any idea what could be wrong?

**Comment by bjrmatos** [comment]
what version of node are you using?

**Comment by bjrmatos** [comment]
you have electron installed in your project, right? what version of electron are you using?

**Comment by bjrmatos** [comment]
i can reproduce the error in node `6.9.2` with electron `1.6.10`, i will take a look, thnks!

**Comment by bjrmatos** [comment]
fixed in `v2.5.2`
